Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person.hs

```diff
@@ -513,3 +513,13 @@ updateUsedCoins (Id driverId) usedCoinValue = updateWithKV [Se.Set BeamP.usedCoi
 
 updateMerchantOperatingCityId :: (MonadFlow m, EsqDBFlow m r, CacheFlow m r) => Id Person -> Id DMOC.MerchantOperatingCity -> m ()
 updateMerchantOperatingCityId (Id driverId) (Id opCityId) = updateWithKV [Se.Set BeamP.merchantOperatingCityId (Just opCityId)] [Se.Is BeamP.id $ Se.Eq driverId]
+
+updateDeviceTokenByPersonId :: (MonadFlow m, EsqDBFlow m r) => Id Person -> m ()
+updateDeviceTokenByPersonId personId = do
```

Comment:
should we rename the function to `clearDeviceTokenByPersonId`. 
as we are not passing `deviceToken` param

Same on rider side.